### DISCUSSION TITLE
High humidity Co2 sensor

### DISCUSSION THREAD

**Discussion by finom-metall** [theyosh/TerrariumPI#551] [Q&A] [answered]
Hello there,

I'm searching a Co2 that can handle a high humidity of up to 100%.
The ones I found are just for 90% or have problems with condensation.
I don't know how this will play out in reality, so i would be interested in your experiences.

I would like to put a CO2 sensor in my fruiting room for mushrooms where I have a humidity of 80-95%, but I think when the humidifier is running, small drops of water settle on the sensor.

I look forward to your answers.

**Comment by theyosh** [comment, marked as answer]
Hi,

actually, I have no clue. I am afraid you have to google to find hardware that can do this. And maybe I can add it to TerrariumPI. But for now, I have not idea how to do this.
